Add ~tirrel Studio Email apprenticeship (#1355)
urbit/urbit.org: content/grants/apprenticeship-studio-email.md

+++
title = "~tirrel Studio Email"
date = "2022-03-02"

[taxonomies]
grant_type = [ "Apprenticeship" ]
grant_category = [ "App Dev", "Other" ]

[extra]
image = ""
description = ""
reward = "$4000 in WSTR"
mentor = "~novlud-padtyv"
assignee = ""
completed = false
work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=A0139&prefill_Grant+Name=~tirrel%20studio%20email%20capabilities"
+++

## Project Description

> The goal will be to build out the email capabilities of the studio mailer app. Currently it allows sending one-off emails and subscription based emails that are tied to a blog. We want to extend it to include transactional and mailing list based email campaigns, email and site analytics. A stretch goal would be to build out a frontend for managing these things in the studio app.

Reviewer: ```suggestion
The goal will be to build out the email capabilities of the studio mailer app. Currently it allows sending one-off emails and subscription based emails that are tied to a blog. We want to extend it to include transactional and mailing list based email campaigns, email and site analytics. We also want to extend the frontend capabilities to include managing these things in the studio app.
```

## Prerequisites

> Some hoon knowledge would be a plus, but we're happy to take someone from the point of zero knowledge to being able to work on this stuff. JS/React skills would be valuable for the stretch goal of building out the frontend.

Reviewer: ```suggestion
Some hoon knowledge would be a plus, but we're happy to take someone from the point of zero knowledge to being able to work on this stuff. JS/React skills would be valuable for the frontend part.
```

## Time Estimates

> At ~10 hours per week I estimate it would take about a month.

Reviewer: ```suggestion
At ~10 hours per week I estimate it would take about 2 months.
```
